node/src/types/sync_leap.rs

```diff
@@ -1474,11 +1452,11 @@ mod tests {
             .iter()
             .take(2)
             .cloned()
-            .map(|block_header_with_metadata| block_header_with_metadata.block_header)
+            .map(|block_header_with_metadata| block_header_with_metadata.block_header().clone())
```

Comment:
Nit: `block_header_with_metadata` → `signed_block_header`

Reply:
Done in [44babf9](https://github.com/casper-network/casper-node/pull/4067/commits/44babf9322a2cf8eac0abff5e4834a7e931581ed).